README.md

```diff
@@ -37,6 +38,7 @@ as a generic parameter. You also need to configure custom file endings for each
 use bevy::prelude::*;
 use bevy_common_assets::json::JsonAssetPlugin;
 use bevy_common_assets::msgpack::MsgPackAssetPlugin;
+use bevy_common_assets::ron::PostcardAssetPlugin;
```

Comment:
```suggestion
use bevy_common_assets::postcard::PostcardAssetPlugin;
```

Reply:
I figured the AssetSaver would be useful for all formats, but I haven't used this part of Bevy yet. I originally was going to have the postcard example load the msgpack asset and save it out to postcard just to avoid writing the example asset file myself but I wasn't able to get that to work.